packages/web/src/new/utils/analytics/types/Buy.ts

```diff
@@ -0,0 +1,17 @@
+import type { MapActionWithNoParams } from 'new/utils/analytics/types/common';
```

Comment:
Why should we import these type unless they will be used in some other places?

Reply:
you mean why this type is not simply declared in this file? I started Receive page. And other pages (send, swap) will also use this type. So it just for future